.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
+  "tabWidth": 2,
+  "semi": false,
```

Comment:
Why not, let's just keep in mind the inconsistencies it can lead to with IIFEs and destructuring

Reply:
I don't have anything against `"semi": true`. I just went with what was previously defined, I'm ok with changing it back :)

Reply:
I'm used to semis in TS but not against some refreshing change 💨 
So good for me as it is!